tcms/templates/include/tc_executions.html

```diff
@@ -23,10 +23,24 @@ <h2 class="card-pf-title">
                         </div>
 
                         <div class="list-view-pf-body">
-                            <div class="list-view-pf-description">
+                            <div class="list-view-pf-description" style="display: flex; justify-content: space-between;">
                                 <div class="list-group-item-text">
                                     <a href="{% url 'test_plan_url_short' execution.run.plan.pk %}">TP-{{ execution.run.plan.pk }}: {{ execution.run.plan.name }}</a>
                                 </div>
+                                <div class="list-group-item-text">
+                                    <div class="completion-rate-container">{% trans 'Completion rate' %}: <span class="completion-rate"></span> 
+                                    <div class="progress">
+                                        <div class="progress-bar progress-bar-danger" role="progressbar"></div>
+                                        <div class="progress-bar progress-bar-success" role="progressbar"></div>
+                                    </div>
+                                </div>
```

Comment:
Looks like a missing div tag somewhere around here.

I'm not sure what your plan for the UI is but I'd prefer that this progress bar be consistent with other and display the negative/positive parts in a single line.

Reply:
To clarify, see the progress bar at the top of Execution trends.